sdk/src/feature_set.rs

```diff
@@ -851,6 +855,7 @@ lazy_static! {
         (bpf_account_data_direct_mapping::id(), "use memory regions to map account data into the rbpf vm instead of copying the data"),
         (last_restart_slot_sysvar::id(), "enable new sysvar last_restart_slot"),
         (reduce_stake_warmup_cooldown::id(), "reduce stake warmup cooldown from 25% to 9%"),
+        (feature_gate_program::id(), "control feature activations with an on-chain program #32783"),
```

Comment:
to improve accuracy of the change

```suggestion
        (programify_feature_gate_program::id(), "move feature gate activation logic to an on-chain program #32783"),
```